### PR TITLE
Fixes #21572 - Send correct Tracer helpers to RemoteEX

### PR DESCRIPTION
**Pull request by sean797**

### CODE REVIEW

File: engines/bastion_katello/app/assets/javascripts/bastion_katello/content-hosts/content/content-host-traces.controller.js [hunk 1]
```diff
@@ -42,9 +42,15 @@ angular.module('Bastion.content-hosts').controller('ContentHostTracesController'
 
         $scope.selectedTraceHelpers = function () {
             var traceHelpers = [], selected = $scope.table.getSelected();
-            angular.forEach(selected, function (value) {
-                traceHelpers.push(value.helper);
-            });
+            for(var value of selected) {
```
Comment (jlsherrill) [inline engines/bastion_katello/app/assets/javascripts/bastion_katello/content-hosts/content/content-host-traces.controller.js]: Looks like eslint doesn't like this:

```

00:06:41.768 app/assets/javascripts/bastion_katello/content-hosts/content/content-host-traces.controller.js
00:06:41.768   45:27  error  Unexpected identifier
```

**Comment by sean797** [comment]
Updated @jlsherrill, I couldn't get it to work with another way as `forEach` and others doesn't allow for a early `return`. I also tried with `find` & `map` but couldn't get it to work.

Hopefully we don't mind the rule? ping @danseethaler as I saw your email to f-dev regarding these rules :)

**Comment by sean797** [comment]
Okay that rule didn't seem to do the trick... not sure why its failing 👎

**Comment by danseethaler** [comment]
It looks to me like it's failing because eslint doesn't know about the `angular` global. We probably need to add a top level `.eslintignore` file to exclude everything except for the `webpack` folder since linting is handled separately under the `bastion_katello` directory.

I'll create a PR for that.

**Comment by danseethaler** [comment]
Just waiting to get the build fixed on #7054 and that should fix this as well.

**Comment by sean797** [comment]
Rebased and tests are failing still, any suggests @danseethaler ?

**Comment by danseethaler** [comment]
@sean797 yeah it looks like your issue is the syntax [here](https://github.com/Katello/katello/pull/7052/files#diff-820df36e89c5758c418377e5c0d63bb8R45) is not supported because we're not yet using babel as the eslint engine. We're working on getting it integrated but ran into an issue with Bastion's `grunt eslint` task when it was on. See [this thread](https://github.com/Katello/katello/pull/7054#issuecomment-344696239).

For now I'd just change that to es5 syntax and push.

**Comment by sean797** [comment]
Thanks @danseethaler what kind of syntax will work? `forEach` wouldn't work because the early return. I also tried with `find`, `map` but no luck.

**Comment by danseethaler** [comment]
Maybe something like this would do the trick?

https://github.com/danseethaler/katello/commit/0c534fc91df36aff7fec03320247bf10c3761232

**Comment by danseethaler** [comment]
@sean797 gotta remove that `;` from [here](https://github.com/Katello/katello/pull/7052/files#diff-820df36e89c5758c418377e5c0d63bb8R61).

**Comment by sean797** [comment]
Thanks @danseethaler that should do the trick 👍

**Pull request merged**